app/js/components/manageApps/ManageApps.jsx

```diff
@@ -157,10 +158,25 @@ export default class ManageApps extends React.Component {
       const apiBaseUrl = `/${applicationDistribution}/${url}/ws/rest`;
       this.requestUrl = '/v1/module/?v=full';
       axios.get(`${urlPrefix}/${apiBaseUrl}${this.requestUrl}`).then(response => {
+        response.data.results.forEach(data => {
+          installedModules.push({
+            appDetails: data,
+            appType: "module",
+            install: false
+          });
+        });
+        installedAddons = installedOwas.concat(installedModules);
         return Promise
           .all(response.data.results.map(data => axios.get(`${ApiHelper.getAddonUrl()}/${data.packageName}`)
           .then(response => response.data)
-          .catch((error) => data)))
+          .catch(error => {
+            this.setState({
+              appList: installedAddons,
+              staticAppList: installedAddons,
+              searchComplete: true
+            });
```

Comment:
This will be deprecated in the the future version of React (as from V16 onwards) I know it's not a big issue but consider using functional setState instead 
`this.setState((prevState, nextProps) => ({
   appList: .....
}))`

Reply:
Thanks for that